src/extension.ts

```diff
@@ -34,11 +34,11 @@ async function configureVersionManager(
   context: vscode.ExtensionContext,
   versionManager: VersionManager
 ) {
-  versionManager.on('IsEvergreen', status => {
+  versionManager.on('IsEvergreen', (status) => {
     vscode.window.showInformationMessage(
       status
-        ? 'Project is already Evergreen. 🌲 Good job!'
-        : 'Angular is out-of-date! Run 🌲 Quick Command > Update Angular to upgrade.'
+        ? 'Project is Evergreen 🌲 Good job!'
+        : 'Angular is out-of-date 😔 Run Quick Command > Update Angular to upgrade.'
```

Comment:
@duluca the Quick Command pane does not contain a command with the actual text `Update Angular` but rather with `Update Angular with ng`. Should we change either one or the other so that it better matches the UI?